tools/docs/check_documentation.sh

```diff
@@ -21,11 +21,21 @@ export KAFKA_EXAMPLES="$OCKAM_HOME/examples/rust/ockam_kafka/examples"
 export E2E_DOCS="$OCKAM_HOME/documentation/use-cases/end-to-end-encryption-with-rust"
 export E2E_EXAMPLES="$OCKAM_HOME/examples/rust/get_started/examples" # TODO
 
+# documentation/use-cases/secure-remote-access-tunnels
 export INLET_DOCS="$OCKAM_HOME/documentation/use-cases/secure-remote-access-tunnels"
 export INLET_EXAMPLES="$OCKAM_HOME/examples/rust/tcp_inlet_and_outlet/examples"
 
+# documentation/use-cases/end-to-end-encrypt-all-application-layer-communication
+export E2EAALC_DOCS="$OCKAM_HOME/documentation/use-cases/end-to-end-encrypt-all-application-layer-communication"
+export E2EAALC_EXAMPLES="$OCKAM_HOME/examples/rust/tcp_inlet_and_outlet/examples"
+
+# documentation/use-cases/run-ockam-on-riscv
+# This documentation is not using compiled code, it could drift from the current Ockam API.
```

Comment:
We can create a script that runs readme code with the following steps
- Run all bash commands by searching for ```bash tags
- Search for ```rust tag and `cargo run` code 

We have delimiters \`\`\` so we know when to stop copying codes/command for run. Thoughts @mrinalwadhwa @neil2468